Makefile

```diff
@@ -398,6 +398,8 @@ project/validate:
 	[ "$$(ls -A $(PWD))" ] || (echo "Project root folder is empty. Please confirm docker has been configured with the correct permissions" && exit 1)
 	@ echo "[$@]: Target test folder validation successful"
 
-install: shellcheck/install bats/install black/install pylint/install pylint-pytest/install pydocstyle/install ec/install yamllint/install cfn-lint/install yq/install bumpversion/install
+install: black/install pylint/install pylint-pytest/install pydocstyle/install ec/install yamllint/install cfn-lint/install yq/install bumpversion/install
+
+install: $(BIN_DIR)
```

Comment:
This change is not necessary, since we dealt with it more appropriately by adding the `$(BIN_DIR)` dependency to the `install/pip/%` target.

Reply:
Did you mean I should remove both line 401 and 403? Or just 403?

Reply:
@lorengordon I'm assuming just line 403, but double checking

Reply:
Correct, just the addition of `install: $(BIN_DIR)`. Specifying `$(BIN_DIR)` as a dependency is better handled on the target `install/pip/%`.